bundler/lib/dependabot/bundler/file_updater/lockfile_updater.rb

```diff
@@ -266,6 +279,7 @@ def evaled_gemfiles
             .reject { |f| f.name.end_with?(".specification") }
             .reject { |f| f.name.end_with?(".lock") }
             .reject { |f| f.name.end_with?(".ruby-version") }
+            .reject { |f| f.name.end_with?(".tool-versions") }
```

Comment:
Perceived complexity check is triggered on this method.

One interesting thing about this is that this and previous lines are redundant, as the last line is `.reject(&:support_file?)` and both `.ruby-version` and `.tool-versions` are marked as support files.

Reply:
Here is my proposal: https://github.com/kpumuk/dependabot-core/commit/3e370f1b8dadeab8048fb79de5beca2e3eaa3abf

Please let me know if this is the preferred approach, and I will merge it into this MR.